packages/ui/package.json

```diff
@@ -46,7 +46,8 @@
     "@types/lodash": "^4.14.161",
     "cleave.js": "^1.6.0",
     "colortranslator": "^1.9.2",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "vue-router": "^4.2.4"
```

Comment:
Why would we need vue-router?

Reply:
This was only added because book-main.ts requires it and it was missing causing the build to fail.

Reply:
You're right. Vue book requires vue-router to work, but not `vuestic-ui` package. We should move `vue-router` from `/docs` to `/ui`, and it must be dev-dependency because it used only in our demos.